### DISCUSSION TITLE
Doc Generator

### DISCUSSION THREAD

**Discussion by SterlingSavariau** [thuliteio/doks#633] [Ideas]
I just started using Sphinx, but I adore this theme so much. In a perfect world, in my head, I would like to use this theme, regardless if it's Hugo or Sphinx, but it also has the feature of downloading the Doc in PDF or Ebook.

Maybe one day.

**Comment by h-enk** [comment]
There is an interesting [Hugo epub theme](https://github.com/weitblick/epub), which you could use next to Doks — I will try this out myself and report my findings here.
